applications/SystemIdentificationApplication/python_scripts/sensor_sensitivity_solvers/system_identification_static_analysis.py

```diff
@@ -22,6 +22,8 @@ def Initialize(self):
             "perturbation_size"            : 1e-8,
             "adapt_perturbation_size"      : true,
             "list_of_sensors"              : [],
+            "aggregation_technique"        : "square_sum",
+            "p_coefficient"                : -1,
```

Comment:
Do we need these two? If we use `'p_coefficient = 1`, then it is basically the `suqare_sum` ryt? I would simplify this.